src/api/auth.ts

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios'
+import type {
+  LoginRequest,
+  LoginResponse,
+  SignUpRequest,
+  SignUpResponse,
+  SetPinRequest,
+  SetPinResponse,
+} from '@/types/auth'
+
+const api = axios.create({
+  baseURL: import.meta.env.VITE_API_BASE_URL,
+  withCredentials: true,
+})
+
+// 로그인
+export const login = async (payload: LoginRequest): Promise<LoginResponse> => {
+  const { data } = await api.post<LoginResponse>('/api/auth/login', payload)
+  return data
+}
```

Comment:
`src/api/api.ts` 공통으로 활용할 수 있는 axios가 정리되어 있어서 그거 활용하면 좋을 것 같아요!!
밑에 회원가입, PIN 설정도 동일하게 바꿔주시면 감사하겠습니다!!

```suggestion
import { post } from '@/api/api'

// 로그인
export const login = async (payload: LoginRequest): Promise<LoginResponse> => {
  const { data } = await post<LoginResponse>('/api/auth/login', payload)
  return data
}
```